CATALOG.md

```diff
@@ -65,7 +65,7 @@ Best Practice Reference|[CNF Best Practice V1.2](https://connect.redhat.com/site
 Property|Description
 ---|---
 Version|v1.0.0
-Description|http://test-network-function.com/testcases/affiliated-certification/container-is-certified tests whether container images have passed the Red Hat Container Certification Program (CCP).
+Description|http://test-network-function.com/testcases/affiliated-certification/container-is-certified tests whether container images listed in tnf_config.yml have passed the Red Hat Container Certification Program (CCP).
```

Comment:
This file is auto generated. The source is the identifiers.go under test-network-functions. Also, it shouldn't use the actual config file name as it can be overridden by env var.

Reply:
And I approved this... sorry.

Reply:
Ouch, my fault too :-/.